example.js

```diff
@@ -85,9 +78,15 @@ class OtpRRExample extends Component {
         <Grid>
           <Row className='main-row'>
             <Col sm={6} md={4} className='sidebar'>
-              <DefaultMainPanel LegIcon={MyLegIcon} ModeIcon={MyModeIcon} />
+              {/* TODO: extract the BATCH elements out of CallTakerPanel. */}
+              {otpConfig.datastoreUrl || otpConfig.routingTypes.find(t => t.key === 'BATCH')
```

Comment:
I think I'd prefer for `datastoreUrl` to be named `callTakerDatastoreUrl` so that it indicates that this is used only for the calltaker module.

Reply:
I'm hesitant to do this because this is what it was previously called and it functions for both the call-taker and field-trip modules.